.teamcity/Gradle_Check/configurations/PerformanceTest.kt

```diff
@@ -43,7 +43,7 @@ class PerformanceTest(model: CIBuildModel, type: PerformanceTestType, stage: Sta
                     + listOf("clean distributed${type.taskId}s -x prepareSamples --baselines %performance.baselines% ${type.extraParameters} -PtimestampedVersion -Porg.gradle.performance.branchName=%teamcity.build.branch% -Porg.gradle.performance.db.url=%performance.db.url% -Porg.gradle.performance.db.username=%performance.db.username% -PteamCityUsername=%TC_USERNAME% -PteamCityPassword=%teamcity.password.restbot% -Porg.gradle.performance.buildTypeId=${IndividualPerformanceScenarioWorkers(model).id} -Porg.gradle.performance.workerTestTaskName=fullPerformanceTest -Porg.gradle.performance.coordinatorBuildId=%teamcity.build.id% -Porg.gradle.performance.db.password=%performance.db.password.tcagent%",
                             buildScanTag("PerformanceTest"))
                             + model.parentBuildCache.gradleParameters(OS.linux)
-                    ).joinToString(separator = " ")
+                    ).joinToString(separator = " ").replace("--daemon", "")
```

Comment:
I saw that the `IndividualPerformanceScenarioWorkers` also switch off the daemon:
https://github.com/gradle/gradle/blob/cd96213b082f1ea6720abdf88b0274c304db8f82/.teamcity/Gradle_Check/configurations/IndividualPerformanceScenarioWorkers.kt#L42-L47

Do you know why this has been in place already?

Reply:
You did it... https://github.com/gradle/gradle/commit/3faae675bcb994e3a58b23d1c3f78e5949e961ea
 
And the commit message says it's intended to avoid some TC issue.

Reply:
I just move the `--no-daemon` to a different place. It has been there before my commit.

Reply:
Then I have no idea...https://github.com/gradle/gradle/commit/7e1f8c7a3b95433087aafe4d6cc57290baea29a6 expressly disable daemon for the individual workers, I guess @jjohannes thought it would avoid interfering performance tests but actually performance tests are executed in isolated processes, right?